pkg/descheduler/strategies/node_taint.go

```diff
@@ -56,7 +51,12 @@ func deletePodsViolatingNodeTaints(client clientset.Interface, policyGroupVersio
 			if maxPodsToEvict > 0 && nodePodCount[node]+1 > maxPodsToEvict {
 				break
 			}
-			if !checkPodsSatisfyTolerations(pods[i], node) {
+			if !utils.TolerationsTolerateTaintsWithFilter(
+				pods[i].Spec.Tolerations,
+				node.Spec.Taints,
+				func(taint *v1.Taint) bool { return taint.Effect == v1.TaintEffectNoSchedule },
+			) {
+				klog.V(2).Infof("Not all taints with NoSchedule effect are tolerated after update for Pod %v on node %v", pods[i].Name, node.Name)
```

Comment:
nit: make `Pod` lower case in this log message to make it consistent with the lower case `node`

Reply:
addressed